### PR TITLE
housekeeping: Mention more supported platforms

### PR DESCRIPTION
**Pull request by worldbeater**
**What are the main goals of this change?**

This synchronizes installation instructions with [`README.md`](https://github.com/reactiveui/ReactiveUI). Currently, the [installation instructions](https://www.reactiveui.net/docs/getting-started/installation/) are a bit outdated, we aren't mentioning UWP, WinUI and MAUI on the website.

### CODE REVIEW

**Review by Arhell — APPROVED** [review]

File: input/docs/getting-started/installation/index.md [hunk 1]
```diff
@@ -5,46 +5,59 @@ Order: 10
...
+| Xamarin.Forms     | [ReactiveUI.XamForms][XamDoc]       | [![XamBadge]][Xam]     |
+| Xamarin.Essentials| [ReactiveUI][XamDoc]                | [![CoreBadge]][Core]   |
+| AndroidX (Xamarin)| [ReactiveUI.AndroidX][DroDoc]       | [![DroXBadge]][DroX]   |
+| Xamarin.Android   | [ReactiveUI.AndroidSupport][DroDoc] | [![DroBadge]][Dro]     |
+| Xamarin.iOS       | [ReactiveUI][IosDoc]                | [![CoreBadge]][Core]   |
+| Xamarin.Mac       | [ReactiveUI][MacDoc]                | [![CoreBadge]][Core]   |
+| Tizen             | [ReactiveUI][CoreDoc]               | [![CoreBadge]][Core]   |
+| Blazor            | [ReactiveUI.Blazor][BlazDoc]        | [![BlazBadge]][Blaz]   |
+| Platform Uno      | [ReactiveUI.Uno][UnoDoc]            | [![UnoBadge]][Uno]     |
+| Platform Uno      | [ReactiveUI.Uno.WinUI][UnoWinUiDoc] | [![UnoWinUiBadge]][UnoWinUi] |
```
Comment (glennawatson) [inline input/docs/getting-started/installation/index.md:29]: Should we call out WinUI for this one?

Comment (worldbeater) [reply input/docs/getting-started/installation/index.md]: Removed "Platform Uno" platform title duplicate. Probably worth mentioning all the platform packages, as the Uno Platform doc page doesn't exist yet https://www.reactiveui.net/docs/getting-started/installation/uno-platform Planning to dive into Uno platform specifics in order to create and fill that missing page soon...

Comment (glennawatson) [reply input/docs/getting-started/installation/index.md]: I would just mention Uno winui rather than removing. Different platform

Comment (glennawatson) [reply input/docs/getting-started/installation/index.md]: Eg you pick plain Uno or uno.winui. Not both.

File: input/docs/getting-started/installation/index.md [hunk 2]
```diff
@@ -79,10 +79,10 @@ ReactiveUI packages are now signed by the dotnet foundation. Only builds from th
 
 [Uno]: https://www.nuget.org/packages/ReactiveUI.Uno/
 [UnoBadge]: https://img.shields.io/nuget/v/ReactiveUI.Uno.svg
-[UnoDoc]: https://reactiveui.net/docs/getting-started/installation/uno-platform
+[UnoDoc]: https://platform.uno/blog/getting-started-with-uno-platform-and-reactiveui/
```
Comment (worldbeater) [inline input/docs/getting-started/installation/index.md:82]: Also updated Uno documentation links so they don't point to 404 but point to their blog posts.

File: input/docs/getting-started/installation/index.md (hunk 1 above)
Comment (worldbeater) [reply input/docs/getting-started/installation/index.md]: Ah, thanks! Corrected the content as requested.

**Review by glennawatson — APPROVED** [review]

**Pull request merged**